desc/plotting.py

```diff
@@ -3843,3 +3844,106 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_gammac(eq, rhos=None, alphas=None, num_pitch=None):
+    """Plot the energetic proxy metric γ_c fo a single flux surface.
+
+    Parameters
+    ----------
+    eq : object
+        Equilibrium object containing magnetic field information
+    rhos : array_like or float, optional
+        Flux surface radii. If float, plots single surface.
+        Default: np.linspace(0.1, 1, 10)
+    alphas : array_like, optional
+        Fieldline label values (toroidal angle).
+        Default: np.linspace(0, 2π, 32, endpoint=True)
+    num_pitch : int, optional
+        Number of pitch angle values for bounce integral calculation.
+        Default: 16
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        The figure object containing the plot
+    ax : matplotlib.axes.Axes
+        The axes object for further customization
+
```

Comment:
Can you add an example section (with example output) to the docstring here like the other plotting functions?

Reply:
Note the formatting here: https://github.com/PlasmaControl/DESC/blob/1d4806d280d11722839a73aee74f0888d07eeca1/desc/plotting.py#L2738C1-L2746C1

Also good to include an example output image so people reading the docs can see what it's actually plotting